modules/storages/app/models/storages/storage_file.rb

```diff
@@ -30,14 +32,14 @@ module Storages
   StorageFile = Data.define(
     :id,
     :name,
-    :size,
+    :size, # Integer >= 0
     :mime_type,
-    :created_at,
-    :last_modified_at,
+    :created_at, # Time? DateTime (deprecated)?
+    :last_modified_at, # Time? DateTime (deprecated)?
     :created_by_name,
     :last_modified_by_name,
-    :location,
-    :permissions
+    :location, # Should always start with a '/'
```

Comment:
🟢 There could be a check in the `initialize` method, to check this and `raise` if it doesn't met the criteria.

Reply:
I'm thinking of adding some type of validation somewhere down the line. I played around a bit with adding it, but for now that's not a pressing matter.